cmd/cosign/cli/options/copy.go

```diff
@@ -32,7 +32,7 @@ var _ Interface = (*CopyOptions)(nil)
 func (o *CopyOptions) AddFlags(cmd *cobra.Command) {
 	o.Registry.AddFlags(cmd)
 
-	cmd.Flags().BoolVar(&o.SignatureOnly, "sig-only", true,
+	cmd.Flags().BoolVar(&o.SignatureOnly, "sig-only", false,
```

Comment:
Is this the regression?

Reply:
yup, defaulted to false before migrating to cobra